crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -82,14 +82,42 @@ type ChangeEpochTransaction {
 }
 
 type Checkpoint {
+	"""
+	The digest of the checkpoint
+	"""
 	digest: String!
+	"""
+	Sequence number of the checkpoint
+	"""
 	sequenceNumber: Int!
+	"""
+	Timestamp of the checkpoint
+	"""
 	timestamp: DateTime
+	"""
+	The aggregate authority signature
+	"""
 	validatorSignature: Base64
+	"""
+	The digest of the previous checkpoint
+	"""
 	previousCheckpointDigest: String
+	"""
+	A single commitment of ECMHLiveObjectSetDigest
+	"""
 	liveObjectSetDigest: String
+	"""
+	Tracks the total number of transactions in the network at the time of the checkpoint
```

Comment:
I might be wrong, but I think this is not the total number of transactions but total number of transaction blocks.